### DISCUSSION TITLE
Crontabs

### DISCUSSION THREAD

**Discussion by LuisZapatos** [spatie/laravel-multitenancy#434] [Q&A]
Hello, good afternoon.

It has been unable to create a **crontab**, it always fails me that it does not find the DB.

And if I try the 'set()' method, it tells me that it can't find 'Spatie\Multitenancy\Facades\Tenant', and it's true that I don't have anything in that route.

Greetings and thank you!

**Comment by masterix21** [comment]
Please post your code.
